test/client/getBalances.ts

```diff
@@ -29,86 +29,72 @@ describe('getBalances', function () {
         assertResultMatch(result, responses.getBalances, 'getBalances')
       })
 
-      // it('getBalances - limit', async function () {
-      //   const options = { limit: 3, ledgerVersion: 123456 }
-      //   this.mockRippled.addResponse(
-      //     'account_info',
-      //     rippled.account_info.normal,
-      //   )
-      //   this.mockRippled.addResponse(
-      //     'account_lines',
-      //     rippledAccountLines.normal,
-      //   )
-      //   this.mockRippled.addResponse('ledger', rippled.ledger.normal)
-      //   const expectedResponse = responses.getBalances.slice(0, 3)
-      //   const result = await this.client.getBalances(testcase.address, options)
-      //   assertResultMatch(result, expectedResponse, 'getBalances')
-      // })
+      it('getBalances - limit', async function () {
+        const request = {
+          account: testcase.address,
+          options: {
+            limit: 10,
+          },
+        }
+        this.mockRippled.addResponse(
+          'account_info',
+          rippled.account_info.normal,
+        )
+        this.mockRippled.addResponse(
+          'account_lines',
+          rippledAccountLines.normal,
+        )
+        this.mockRippled.addResponse('ledger', rippled.ledger.normal)
+        const expectedResponse = responses.getBalances.slice(0, 10)
```

Comment:
Could you extract the number 10 in this slice into a variable so that you're showing the limit in `options` above should match the length of this expected result?

Reply:
Actually, you can use the options object itself here :) - 

`const expectedResponse = responses.getBalances.slice(0, request.options.limit)`